diff --git a/playground/_scripts/revendorToolsInternal.bash b/playground/_scripts/revendorToolsInternal.bash
index 771d24ec..ea3def82 100755
--- a/playground/_scripts/revendorToolsInternal.bash
+++ b/playground/_scripts/revendorToolsInternal.bash
@@ -29,7 +29,7 @@ popd > /dev/null
 
 # TODO this is very fragile - we need to move away from this to a proper
 # dependency-based approach to minimal copy
-for i in "" filetypes encoding third_party/yaml encoding/yaml astinternal cueexperiment envflag source
+for i in "" filetypes encoding encoding/yaml astinternal cueexperiment envflag source
 do
 	rsync -a --relative --delete $td/$path@$version/internal/./$i/ ./internal/cuelang_org_go_internal/
 	find ./internal/cuelang_org_go_internal/$i -mindepth 1 -maxdepth 1 -type d -exec rm -rf {} +
diff --git a/content/docs/tutorial/converting-json-schema-to-cue/en.md b/content/docs/tutorial/converting-json-schema-to-cue/en.md
index cccda35d..e216a16e 100644
--- a/content/docs/tutorial/converting-json-schema-to-cue/en.md
+++ b/content/docs/tutorial/converting-json-schema-to-cue/en.md
@@ -128,9 +128,11 @@ package cuisine
 
 #restaurant: {
 	@jsonschema(schema="http://json-schema.org/draft-07/schema#")
-	name!:    string
-	cuisine!: string
-	tables!: [...#table]
+	close({
+		name!:    string
+		cuisine!: string
+		tables!: [...#table]
+	})
 
 	#table: {
 		seats!: >=2 & <=10
