diff --git a/internal/cmd/preprocessor/cmd/testdata/execute_transform_normalise.txtar b/internal/cmd/preprocessor/cmd/testdata/execute_transform_normalise.txtar
index 97553c5cb..ba56c056d 100644
--- a/internal/cmd/preprocessor/cmd/testdata/execute_transform_normalise.txtar
+++ b/internal/cmd/preprocessor/cmd/testdata/execute_transform_normalise.txtar
@@ -28,9 +28,7 @@ content: dir: page: {
 >y: 5
 >}
 >-- out.cue --
->x: {
->    y: 5
->}
+>x: y: 5
 >{{{end}}}
 >
 >This is a {{{ def "test" }}}
@@ -45,9 +43,7 @@ content: dir: page: {
 >	y: 5
 >}
 >-- out.cue --
->x: {
->    y: 5
->}
+>x: y: 5
 >{{{end}}}
 >
 >This is a {{{def "test"}}}
diff --git a/playground/impl_test.go b/playground/impl_test.go
index c62074749..b53ec02c1 100644
--- a/playground/impl_test.go
+++ b/playground/impl_test.go
@@ -80,8 +80,8 @@ var testTable = []testCase{
 			[ID=_]: x: y: ID
 			"foo": {}
 		`,
-		OutVal: "foo: {\n\tx: {\n\t\ty: \"foo\"\n\t}\n}\n",
-	},
+		OutVal: `foo: x: y: "foo"
+`},
 	{
 		In: inputCUE, Fn: functionExport, Out: outputJSON,
 		InVal: `
@@ -98,16 +98,18 @@ var testTable = []testCase{
 			x: string
 			y: (x): string
 		`,
-		OutVal: "x: string\ny: {\n\t(x): string\n}\n",
-	},
+		OutVal: `x: string
+y: (x): string
+`},
 	{
 		In: inputCUE, Fn: functionEval, Out: outputCUE,
 		InVal: `
 			x: x2: "foo"
 			y: x.missing
 		`,
-		OutVal: "x: {\n\tx2: \"foo\"\n}\ny: x.missing\n",
-	},
+		OutVal: `x: x2: "foo"
+y: x.missing
+`},

 	// Simple format checks; no errors
 	{
